packages/components/twin.d.ts

```diff
@@ -1,17 +1,9 @@
 import "twin.macro";
+import "@types/styled-components";
```

Comment:
@anniehu4 I think this addresses the same issue as the piece I deleted below, but I want to check with you since you added it - what should I test to make sure it worked?

Reply:
ah I just grabbed it from the [Twin guide](https://github.com/ben-rogerson/twin.examples/tree/master/react-styled-components#complete-the-typescript-setup). I think if there's no typescript error on the Heading component (or anywhere we pass `as` to twin), that means it's working